src/stringClass.js

```diff
@@ -1,75 +1,75 @@
 const stringClass = {
   /**
    * This method takes a string and tests if it contains vowels
-   * @param {String} this
+   * It extends the String.prototype to achieve this
    * @return {Boolean} true or false
    */
   hasVowels() {
-    return /[aeiou]/.test(this);
+    return /[aeiou]/i.test(this);
   },
 
-/**
-   * This method takes a string and returns it in uppercase
-   * @param {String} this
-   * @return {String} this
-   */
+  /**
+     * This method takes a string and returns it in uppercase
+     * It extends the String.prototype to achieve this
```

Comment:
`It extends the String.prototype to achieve this` this looks redundant.
Let's take it out.